test/development/acceptance/hydration-error.test.ts

```diff
@@ -38,7 +37,7 @@ describe.skip('Error overlay for hydration errors (React 18)', () => {
     await session.assertHasRedbox()
 
     expect(await session.getRedboxDescription()).toMatchInlineSnapshot(`
-      "Error: Text content does not match server-rendered HTML.
+      "Text content does not match server-rendered HTML.
```

Comment:
Slight change in behavior but showing `Error:` in an error overlay feels a bit redundant